Makefile

```
@@ -495,7 +495,8 @@ uninstall:
 		"$(DEST)/bin/kubectl.lima" \
 		"$(DEST)/share/man/man1/lima.1" \
 		"$(DEST)/share/man/man1/limactl"*".1" \
-		"$(DEST)/share/lima" "$(DEST)/share/doc/lima"
+		"$(DEST)/share/lima" "$(DEST)/share/doc/lima" \
+		"$(DEST)/libexec/lima"
```

I wonder if deleting all of `/usr/local/libexec/lima` is the right thing.

I assumed external drivers that are not bundled by Lima would also be installed there. In which case deleting them would be wrong.

Same thing for additional plugins. Can they also be installed in `/usr/local/libexec/lima`? If they can, then again, we should not wipe them.

So to be safe we should just delete:

```suggestion
		"$(DEST)/libexec/lima/lima-driver-qemu$(exe)" \
		"$(DEST)/libexec/lima/lima-driver-vz$(exe)" \
		"$(DEST)/libexec/lima/lima-driver-wsl$(exe)" \
		"$(DEST)/libexec/lima/limactl-mcp$(exe)"
```